src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -60,35 +67,18 @@ const TopOffset = styled.View`
   align-items: center;
 `
 
-const CenteringContainer = styled.View`
-  flex: 1;
-  justify-content: center;
-  align-items: center;
-`
-
 const Icon = styled.Image`
   width: 20px;
   height: 20px;
   margin: 10px;
 `
 
-const StyledText = styled.Text`
-  margin: 10px;
-  font-family: ${Fonts.GaramondRegular};
-  font-size: 16px;
-  text-align: center;
-  color: #666666;
-`
+const CenteringContainer = props => <Flex flex={1} justifyContent="center" alignItems="center" {...props} />
```

Comment:
I imagine something like this would be exported from the Container element?

Reply:
Absolutely. I left it here since this is the only place we use it today.